version.txt

```diff
@@ -0,0 +1 @@
+2.4.5
```

Comment:
Done. I plan to use the contents of this file more in the future, for example this will be used in `scripts/ci/install-catalog-source.sh` to initiate the `VERSION` field.